src/bootstrap/native.rs

```diff
@@ -252,8 +252,13 @@ impl Step for Llvm {
             // FIXME: if the llvm root for the build triple is overridden then we
             //        should use llvm-tblgen from there, also should verify that it
             //        actually exists most of the time in normal installs of LLVM.
-            let host = builder.llvm_out(builder.config.build).join("bin/llvm-tblgen");
-            cfg.define("CMAKE_CROSSCOMPILING", "True").define("LLVM_TABLEGEN", &host);
+            let host = builder.llvm_out(builder.config.build);
+            let extension = if builder.config.build.contains("windows") { "exe" } else { "" };
+
+            cfg.define("CMAKE_CROSSCOMPILING", "True");
+            cfg.define("LLVM_TABLEGEN", host.join("bin/llvm-tblgen").with_extension(extension));
+            cfg.define("LLVM_NM", host.join("bin/llvm-nm").with_extension(extension));
+            cfg.define("LLVM_CONFIG_PATH", host.join("bin/llvm-config").with_extension(extension));
```

Comment:
Nit: you can use the [`std::env::consts::EXE_EXTENSION`](https://doc.rust-lang.org/std/env/consts/constant.EXE_EXTENSION.html) constant to get the exe suffix for the current host platform.

Reply:
Nice!